src/airflow/providers/anomalo/example_dags/example_anomalo.py

```diff
@@ -37,7 +37,7 @@
 
     anomalo_validate = AnomaloPassFailOperator(
         task_id="AnomaloPassFail",
-        table_name="public-bq.crypto_bitcoin.outputs",
+        table_name="public-bq.austin_bikeshare.bikeshare_stations",
```

Comment:
Just for testing; this runs significantly faster (nigh-instananeously) instead of `crypto_bitcoin.outputs`, which takes > 15 mins